ifStatement.js

```diff
@@ -34,6 +41,17 @@ Steps:
 ***********************************************************/
 // TODO: ADD YOUR CODE BELOW
 
+
+const userName ="mohamed";
+let userTest = prompt("enter user name");
+const role ="Admin";
+if(userName.toLowerCase() ===userTest && role.toLowerCase() === "admin" ){
```

Comment:
You have to lowercase the user input `userText,` not the constant name `userName`
No need to lowercase the role